source/loader/layers/validation/ur_valddi.cpp

```diff
@@ -9,7 +9,51 @@
  */
 #include "ur_validation_layer.hpp"
 
+#include <atomic>
+#include <functional>
+#include <map>
+#include <mutex>
+#include <shared_mutex>
+#include <utility>
+
 namespace validation_layer {
+struct {
+    std::map<void *, std::atomic<int>> counts;
```

Comment:
Why std::map and not std::unordered_map?

Reply:
I've changed it to std::unordered_map.